stdlib/public/SwiftShims/RefCount.h

```diff
@@ -369,33 +388,56 @@ class RefCountBitsT {
   enum Immortal_t { Immortal };
 
   LLVM_ATTRIBUTE_ALWAYS_INLINE
-  bool isImmortal() const {
-    return bool(getField(IsImmortal));
+  bool isImmortal(bool checkSlowRCBit) const {
```

Comment:
Since this is always called with a constant `true` or `false`, it might be clearer to just split it into two functions.